src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "user123";
+    private static final String INVALID_LOGIN = null;
+    private static final String VALID_PASSWORD = "password123";
+    private static final String INVALID_PASSWORD = "nvdf";
+    private static final int VALID_AGE = 20;
```

Comment:
also better to test with edge cases
```suggestion
    private static final int VALID_AGE = 18;
```